data_preprocessing/main.py

```diff
@@ -0,0 +1,28 @@
+from data_preprocessing.pandas_data_preprocessor import PandasDataPreprocessor
+
+
+def main():
+    columns_to_filter = ["TotPkts", "TotBytes", "SrcBytes", "flag", "service", "count", "dst_bytes", "class"]
+    columns_to_transform_to_log = ["TotPkts", "SrcBytes", "dst_bytes"]
+
+    data_processor = PandasDataPreprocessor(data_type='IDS_Data1')
+
+    data_processor.load_data('../data/IDS_Data1.csv')
+    data_processor.check_missing_values()
+    data_processor.show_basic_statistics()
+
+    data_processor.choose_columns(columns_to_filter)
+    for column in columns_to_filter:
+        data_processor.plot_histogram(column)
+
+    for column in columns_to_transform_to_log:
+        data_processor.transform_column_data_to_logarithmic_scale(column)
+        data_processor.plot_histogram(column)
```

Comment:
:question:  For some data, I decided to transform it to log scale instead of removing values as we first talk about it. As I read about it I think it will be better to perform such transformation since machine learning models may be sensivite for such outliers. What you guys think about it?

Reply:
Good choice, I suppose that should help us to build better model.  Apart from that we can always try to generate second dataset and see what is better